.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+
+Example:
+```bash
+if [[ ! -d /opt/snipe-it ]]; then
+    msg_error "No ${APP} Installation Found!"
+    exit
+  fi
+```
+### 3.3 **Check Version**
+- Befor updating, check if a new Version exists. 
+- For this we use the `${APPLICATION}_version.txt` file created in `/opt` during the install.
+
+Example with a Github Release:
+```bash
+ RELEASE=$(curl -fsSL https://api.github.com/repos/snipe/snipe-it/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+  if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+    msg_info "Updating ${APP} to v${RELEASE}"
+    #DO UPDATE
+  else
+    msg_ok "No update required. ${APP} is already at v${RELEASE}."
+  fi
+  exit
+}
+```
+### 3.4 **Verbosity**
+- Use the appropiate flag (**-q** in the examples) for a command to suppres its output.
+Example:
+```bash
+wget -q
+unzip -q
+```
+- If a command dose not come with such a functionality use `&>/dev/null` to suppress its output.
+
+Example:
+```bash
+php artisan migrate --force &>/dev/null
+php artisan config:clear &>/dev/null
+```
+
+### 3.5 **Backups**
+- Backup userdata if nessesary.
+- Move all userdata back in the Directory when the update is finnished.
+>[!NOTE]
+>This is not meant to be a permantent backup
+
+Example backup:
+```bash
+  mv /opt/snipe-it /opt/snipe-it-backup
+```
+Example config restore:
+```bash
+  cp /opt/snipe-it-backup/.env /opt/snipe-it/.env
+  cp -r /opt/snipe-it-backup/public/uploads/ /opt/snipe-it/public/uploads/
+  cp -r /opt/snipe-it-backup/storage/private_uploads /opt/snipe-it/storage/private_uploads
+```
+
+### 3.6 **Cleanup**
+- Do not forget to remove any temporary files/folders such as zip-files or temporary backups.
+Example:
+```bash
+  rm -rf /opt/v${RELEASE}.zip
+  rm -rf /opt/snipe-it-backup
+```
+
+### 3.7 **No update function**
+- In case you can not provide a update function use the following code to provide user feedback.
+```bash
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+    if [[ ! -d /opt/snipeit ]]; then
+        msg_error "No ${APP} Installation Found!"
+        exit
+    fi
+    msg_error "Ther is currently no automatic update function for ${APP}."
+    exit
+}
+```
+
+---
+
+## 4 **End of the Script**
+- `start`: Launches Whiptail dialogue
+- `build_container`: Collects and integrates user settings
+- `description`: Sets LXC container description
+- With `echo -e "${TAB}${GATEWAY}${BGN}http://${IP}${CL}"` you can point the user to the IP:PORT/folder needed to access the App.
+
+```bash
+start
+build_container
+description
+
+msg_ok "Completed Successfully!\n"
+echo -e "${CREATING}${GN}${APP} setup has been successfully initialized!${CL}"
+echo -e "${INFO}${YW} Access it using the following URL:${CL}"
+echo -e "${TAB}${GATEWAY}${BGN}http://${IP}${CL}"
+```
+---
+
+## 5. **Best Practices Checklist**
```

Comment:
```suggestion
## 5. **Contribution checklist**
```

These do not seem to be best practices. These seem to be musts for contributions.